chcemvediet/apps/inforequests/tests/test_models/test_action.py

```diff
@@ -730,10 +800,26 @@ def test_order_by_effective_date_query_method(self):
                 u'2015-10-05',
                 ]
         random.shuffle(dates)
-        inforequest = self._create_inforequest()
-        branch = self._create_branch(inforequest=inforequest)
         actions = []
         for date in dates:
-            actions.append(self._create_action(branch=branch, effective_date=naive_date(date)))
-        result = Action.objects.order_by_effective_date()
-        self.assertEqual(list(result), sorted(actions, key=lambda a: (a.effective_date, a.pk)))
+            actions.append(self._create_action(created=naive_date(date), subject=u'order_by'))
+        result = Action.objects.filter(subject=u'order_by').order_by_created()
+        self.assertEqual(list(result), sorted(actions, key=lambda a: (a.created, a.pk)))
+
+    def test_before_and_after_query_methods(self):
+        Action.objects.all().delete()
```

Comment:
@martinmacko47 Napadaju ma tri moznosti na riesenie tochto problemu. 
a) Action.objects.all().delete()
b) zadefinovat field napr. `content` a `Action.objects.filter(conten=u'tested_actions')`
c) pri vytvarani actions pomocou `_create_action` vytvorit vlastny inforequest a selectnut testovane actions s `Action.objects.of_inforequest(inforequest=inforequest)`

Pozmenit taketo selectovanie oproti tomu ako to je spravene teraz?

Reply:
@viliambalaz `Action.objects.all().delete()` IMHO je fajn